src/session/session_api.c

```diff
@@ -1291,7 +1291,7 @@ __session_drop(WT_SESSION *wt_session, const char *uri, const char *config)
         WT_STAT_CONN_INCR(session, session_table_drop_success);
 
     /* Note: drop operations cannot be unrolled (yet?). */
-    API_END_RET_NOTFOUND_MAP(session, ret);
+    API_END_RET_NOTFOUND_MAP_OK(session, ret, EBUSY);
```

Comment:
This change related to EBUSY is unrelated to ENOENT and this ticket AFAICT. Why is it necessary? In this form it is not very intuitive. The macro name talks about NOTFOUND mapping and the error in the args is EBUSY.